tasks/DigitalOutMultiplexerTask.hpp

```diff
@@ -0,0 +1,117 @@
+/* Generated from orogen/lib/orogen/templates/tasks/Task.hpp */
+
+#ifndef MOTORS_ROBOTEQ_CANOPEN_DIGITALOUTMULTIPLEXERTASK_TASK_HPP
+#define MOTORS_ROBOTEQ_CANOPEN_DIGITALOUTMULTIPLEXERTASK_TASK_HPP
+
+#include "linux_gpios/linux_gpiosTypes.hpp"
+#include "motors_roboteq_canopen/DigitalOutMultiplexerTaskBase.hpp"
+#include "motors_roboteq_canopen/motors_roboteq_canopenTypes.hpp"
+#include "base/Time.hpp"
+#include <optional>
```

Comment:
I did not pay attention in the first review, but you just need these here, the rest must go in the cpp
```suggestion
#include "motors_roboteq_canopen/DigitalOutMultiplexerTaskBase.hpp"
#include <optional>
```

Reply:
I do need motors_roboteq_canopenTypes.hpp, but I like this kind of change. Done.